### PR TITLE
add misc-cmds

### PR DESCRIPTION
**Pull request by asdfugil**

### CODE REVIEW

File: makefiles/misc-cmds.mk [hunk 1]
```diff
@@ -0,0 +1,51 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += misc-cmds
+MISC-CMDS_VERSION := 34
+DEB_MISC-CMDS_V   ?= $(MISC-CMDS_VERSION)-1
```
Comment (CRKatri) [inline makefiles/misc-cmds.mk:7]: `-1`?

File: makefiles/misc-cmds.mk [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin
+	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/{man/man1,misc}
+	@# tsort conflics with coreutils
+	cd $(BUILD_WORK)/misc-cmds; \
+	for bin in calendar leave ncal units; do \
+		$(CC) $(LDFLAGS) $(CFLAGS) -o $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/$$bin $$bin/*.c -lncurses; \
+		cp $(BUILD_WORK)/misc-cmds/$$bin/$$bin.1 $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man/man1/$$bin.1 ;\
+	done
+	cp -a $(BUILD_WORK)/misc-cmds/calendar/calendars $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/calendar
+	cp -a $(BUILD_WORK)/misc-cmds/units/units.lib $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/misc
```
Comment (CRKatri) [inline makefiles/misc-cmds.mk:31]: You may need to patch the commands to work if the prefix isn't /usr

Comment (asdfugil) [reply makefiles/misc-cmds.mk:31]: Which I did

Comment (CRKatri) [reply makefiles/misc-cmds.mk:31]: :man_facepalming:

**Review by CRKatri — APPROVED** [review]

File: makefiles/misc-cmds.mk [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	$(SED) -i '1 i\typedef unsigned int u_int;' $(BUILD_WORK)/misc-cmds/leave/leave.c
+	$(SED) -i "s|/usr|$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)|g" $(BUILD_WORK)/misc-cmds/{calendar,units}/pathnames.h
+
+ifneq ($(wildcard $(BUILD_WORK)/misc-cmds/.build_complete),)
+misc-cmds:
+	@echo "Using previously built misc-cmds."
+else
+misc-cmds: misc-cmds-setup ncurses
+	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin
+	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/{man/man1,misc}
```
Comment (TheRealKeto) [inline makefiles/misc-cmds.mk:23]: These can be done in line 12, in one line.
```suggestion

```

File: makefiles/misc-cmds.mk [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+endif
+
+SUBPROJECTS       += misc-cmds
+MISC-CMDS_VERSION := 34
+DEB_MISC-CMDS_V   ?= $(MISC-CMDS_VERSION)
+
+misc-cmds-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://opensource.apple.com/tarballs/misc_cmds/misc_cmds-$(MISC-CMDS_VERSION).tar.gz
+	$(call EXTRACT_TAR,misc_cmds-$(MISC-CMDS_VERSION).tar.gz,misc_cmds-$(MISC-CMDS_VERSION),misc-cmds)
+	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin
```
Comment (TheRealKeto) [inline makefiles/misc-cmds.mk:12]: Create directories from lines 22-23.
```suggestion
	mkdir -p $(BUILD_STAGE)/misc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/{bin,share/{man/man1,misc}}
```

**Review by TheRealKeto — CHANGES_REQUESTED** [review]
Unsure how someone missed this one...

**Pull request merged**